istio-control/istio-autoinject/values.yaml

```diff
@@ -4,52 +4,14 @@ sidecarInjectorWebhook:
   # retry.
   replicaCount: 1
 
+  image: sidecar_injector
```

Comment:
Let's remove this - default is provided, not something we expect users to set. 
Not clear why it was even added.

Reply:
I think its so you can provide a full image like `image: myrepo/injector:tag`. We need to change this in pilot (maybe others) too if we remove this so they are consistent

Reply:
Eventually we want to move to kustomize syntax.
```
images:
  - name: postgres
    newName: my-registry/my-postgres
    newTag: v1
  - name: nginx
    newTag: 1.8.0
  - name: my-demo-app
    newName: my-app
  - name: alpine
    digest: sha256:24a0c4b4a4c0eb97a1aabb8e29f18e917d05abfe1b7a7c07857230879ce7d3d3

```

Reply:
Which implies the image 'base' name will be fixed in the templates, and user can override it using kustomize. 
We can keep the template and allow image to override, for backward compat - but a missing image in values.yaml should be supported, and we can start deprecating it.